src/app/(proper_react)/redesign/(authenticated)/user/dashboard/DashboardTopBanner.tsx

```diff
@@ -54,13 +61,52 @@ export const DashboardTopBanner = (props: DashboardTopBannerProps) => {
   );
 
   const contentData = {
+    ScanInProgressContent: {
```

Comment:
Hoo boy, this is going to conflict so hard with https://github.com/mozilla/blurts-server/pull/3405 :cry:

Reply:
It did conflict very hard. Should be resolved now. :)